ManatiCR/manatime
# 48: Create node/add/* aliases.

**Steps to test**
- Check if each one of the followings paths are working:
  - clients/add
  - projects/add
  - tasks/add
  - users/add
- Verify the manatime_feature_config was created right.
- Check the comments of the . install.

[Issue link](https://github.com/ManatiCR/manatime/issues/48)
